pom.xml

```diff
@@ -35,6 +35,7 @@
 
   <properties>
     <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
+    <maven.jar.plugin>3.2.0</maven.jar.plugin>
```

Comment:
What's wrong with the version inherited from the parent?

Reply:
Ah, I thought it wasn't specified in the parent. I just found that it's `3.1.2` so I'll delete this.